README.md

```diff
@@ -1,4 +1,54 @@
 # BSC-MEV-Sentry
 
-BSC-MEV-Sentry is a proxy service within the BSC MEV architecture designed to shield the validator network.
-It functions to forward requests and pay to builders.
+BSC-MEV-Sentry serves as the proxy service for BSC MEV architecture, It has the following features:
+
+1. Forward RPC requests under mev namespace to validators/builders.
+2. Pay builders for their bid.
+3. Monitor validators' status and health.
+
+See also: https://github.com/bnb-chain/BEPs/pull/322
+
+# Usage
+
+1. `make build`
+2. `.build/sentry -config ./configs/config.toml`
+
+Sentry settings are configured in the `config.toml` file. The following is an example of a `config.toml` file:
+
+```
+[Service]
+HTTPListenAddr = "localhost:8555" # The address to listen on for HTTP requests.
+RPCConcurrency = 100 # The maximum number of concurrent requests.
+RPCTimeout = "10s" # The timeout for RPC requests.
+
+[[Validators]] # A list of validators to forward requests to.
+PrivateURL = "https://bsc-fuji" # The private rpc url of the validator, it can only been accessed in the local network.
+PublicHostName = "bsc-fuji" # The domain name of the validator, if a request's HOST info is same with this, it will be forwarded to the validator.
+PayAccountMode = "privateKey" # The unlock mode of the pay bid account.
```

Comment:
Is it possible that a validator dont accept any bid with fee, how can such a validator set the `PayAccountMode`?

Reply:
pay bid tx is a must